client/src/services/plugins/ExcelImportPlugin.js

```diff
@@ -32,7 +34,9 @@ class StandardExcelImportFormat {
     const errors = [];
     for (let i = 1; i < resp.rows.length; i++) {
       const row = resp.rows[i];
-
+      if (row.length === 0) {
+        break;
```

Comment:
Is break the right option here? Or should we `continue;`? Or generate an exception so we can inform the user about what happened?

Reply:
The specific problem with this line was actually a bug I found in the Code. Since I don't have Office installed I use Google spreadsheets to change the file for testing purposes and find out Google always insert an empty row at the end of the file.
First approach was enclose the code in the IF, that way we will be safe in either case empty row at EOD or user missing a line and trying to upload a file with and empty line, but cannot compile this due to a config in eslint 
![image](https://user-images.githubusercontent.com/97867829/156634223-f99933c3-3478-4597-baa3-bdf5c387c4d7.png)
I'm changing it to continue; to enable the user to include empty lines now.